flipp/src/app/components/AppContainer.js

```diff
@@ -1,7 +1,31 @@
+"use client";
 import CardContainer from "./CardContainer";
 import Header from "./Header";
 import ButtonContainer from "./ButtonContainer";
+import { useState } from "react";
 export default function AppContainer() {
+  const [flip, setFlip] = useState(false);
```

Comment:
We have to keep track if the card is flipped with this variable here that's being set by the onClick event handler on the "flip" button.